README.md

```diff
@@ -45,17 +45,23 @@ commands.
 
 ## Parallel Tests
 
-The test reporter supports aggregating multiple test coverage reports from, for example, parallel runs
-together into a single report. After aggregation, the test reporter can send up the single aggregated
-report to Code Climate.
+Support for parallel tests is made possible by using sub-commands provided by the test
+reporter.  Specifically, the test reporter has sub-commands to:
 
-This requires you store the partial payloads yourself after each test, then
-download them to one location before using the reporter to upload a summed
-payload.
+1. format partial results (`format-coverage`)
+1. aggregate partial results into a single result (`sum-coverage`) and
+1. upload the single result to Code Climate (`upload-coverage`)
+
+To make use of these commands, parallel test support requires:
+
+1. the ability to run commands after *each* individual worker has completed (most CI systems support this)
```

Comment:
FWIW: I intentionally used the phrase "after each _batch of tests_" to avoid the "worker"/"container" confusion.